modules/editor-events/module.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Elementor\Modules\EditorEvents;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Tracker;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+
+	public function __construct() {
+		parent::__construct();
+
+		add_action( 'elementor/editor/localize_settings', function( array $settings ) {
```

Comment:
Is this a filter hook, not an action hook. Please confirm?

Reply:
Removed at all